packages/vaults/src/transaction/config/vault_naming.ts

```diff
@@ -9,8 +9,22 @@ import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
 
+/**
+ * Transaction for updating the name/description of the vault.
+ * Can be requested/approved only by users with the admin role.
+ * This transaction can be executed in a batch,
+ * which means either all transactions marked with a single batch_id
+ * are executed or rejected simultaneously.
```

Comment:
@olsemeno it will be helpful to describe the conditions under which the batch will be executed or rejected. Is it rejected only if one of the transactions fails, and executes otherwise?